docs/cli_installation.md

```diff
@@ -49,24 +55,7 @@ brew install argocd
 
 ### Download With Curl
 
-You can view the latest version of Argo CD at the link above or run the following command to grab the version:
-
-```bash
-VERSION=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
-```
-
-Replace `VERSION` in the command below with the version of Argo CD you would like to download:
-
-```bash
-curl -sSL -o argocd-darwin-amd64 https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-darwin-amd64
-```
-
-Install the Argo CD CLI binary:
-
-```bash
-sudo install -m 555 argocd-darwin-amd64 /usr/local/bin/argocd
-rm argocd-darwin-amd64
-```
+See [Linux curl instructions](#download-with-curl).
```

Comment:
The Linux section also has the same instructions for homebrew. So if the intent is to remove this duplication, would it make sense to include Mac in the `## Linux and WSL` heading and then specify the Mac applicable sections in the sub-headings?

Reply:
(could be maybe!) - will leave that up to maintainers to decide